makarov ivan/site_grid.html

```diff
@@ -90,18 +102,20 @@
             .container {
                 width: 581px;
                 height: 438px;
+                margin: auto;   /*главный див будет по центру*/
                 display: grid;
```

Comment:
I really like your approach with grid layout. Also, I like that the widget looks great in all browsers